### PR TITLE
Document the fact that we only support v1.0 in CWL

### PR DESCRIPTION
**Pull request by geoffjentry**
This doesn't solve the underlying problem I ran into yesterday, but it's a start

### CODE REVIEW

File: engine/src/main/resources/swagger/cromwell.yaml [hunk 1]
```diff
@@ -117,10 +117,10 @@ paths:
           in: formData
         - $ref: '#/parameters/workflowTypeParam'
         - name: workflowTypeVersion
-          description: The specification version for the workflow language being used. Cromwell currently supports draft-2.
+          description: The specification version for the workflow language being used. Cromwell currently supports draft-2. For CWL, Cromwell currently support v1.0.
```
Comment (mcovarr) [inline engine/src/main/resources/swagger/cromwell.yaml]: support => supports

File: engine/src/main/resources/swagger/cromwell.yaml [hunk 2]
```diff
@@ -63,10 +63,10 @@ paths:
           type: string
           in: formData
         - name: workflowTypeVersion
-          description: The specification version for the workflow language being used. For WDL, Cromwell currently supports draft-2.
+          description: The specification version for the workflow language being used. For WDL, Cromwell currently supports draft-2. For CWL, Cromwell currently support v1.0.
```
Comment (mcovarr) [inline engine/src/main/resources/swagger/cromwell.yaml]: support => supports

**Comment by gemlam3** [comment]
@geoffjentry anything left to do for this prior to merging?

**Comment by geoffjentry** [comment]
@gemlam3 yes

**Pull request merged**